src/views/Scan/ScanPage.vue

```diff
@@ -110,6 +125,71 @@ function linkName() {
 function resetPage() {
   boards = useBoardsStore();
   board = boards.getNewest();
+}
 
+async function fillAllHands(router: Router){
+  board.northHand = json.direction[0].detections;
+  board.eastHand = json.direction[1].detections;
+  board.southHand = json.direction[2].detections;
+  board.westHand = json.direction[3].detections;
+
+  if (route.path.startsWith("/active-tournament")) {
+    await getDDSForNewestBoard();
+    router.push("/active-tournament/single-table/score");
+  } else {
+    router.push("/scan/game-details");
+  }
 }
+
+function reset() {
+  photoStore.deletePhotos();
+}   
 </script>
+
+
+<style scoped>
+p{
+  font-size: 25px;
+  text-align: center;
+  color: #000;
+  font-family: system-ui;
+}
+
+.light-orange-block{
+  margin: 15px;
+  border: 3px solid #f9a0658f;
+  background-color: #f8965423;
+  border-radius: 5px;
+}
```

Comment:
Zou ik juist een witregel toevoegen
```suggestion
}

```